Added a translation to german

LarsArtmann opened the pull request.

Commit:
Added a translation to german

Crypnotic commented:
The code to load all provided locales from the jar has to be written.

Also, I am not willing to ship a default locale in a language that I cannot translate myself unless someone is willing to update it with new messages as they come out. If you can't do so, I won't accept this

LarsArtmann commented:
Is ok, i update the german locale file if new messages will added

LarsArtmann commented:
I have to do it anyway for my server

Commit:
Update de_DE.conf

LarsArtmann commented:
Added german messages for #30

Commit:
Added german translation for #30

https://github.com/Crypnotic/Neutron/pull/30/commits/6da5899cefc1d2d77dca8c9eba3907893c1abaa1

LarsArtmann commented:
@Crypnotic What is the status of this Pull request?
